webui/react/src/pages/JobQueue/JobQueue.tsx

```diff
@@ -98,30 +101,34 @@ const JobQueue: React.FC<Props> = ({ bodyNoPadding, selectedRp, jobState }) => {
 
   usePolling(fetchAll, { rerunOnNewFn: true });
 
+  const getCommandType = (jobType: Determinedjobv1Type): CommandType | null => {
```

Comment:
Should I move this func to utility or leave it here?

Reply:
nit: do we have a utils/job? let's put it there for discoverability.

Reply:
the same function is already in utils/job!